server/src/document/document.controller.ts

```diff
@@ -1,4 +1,5 @@
-import { Controller,
+import {
```

Comment:
Are changes in this file all linting?

Reply:
uh oh. yep except for the bottom of the file.